packages/devtools_app/lib/src/screens/inspector/inspector_screen.dart

```diff
@@ -565,6 +569,59 @@ class _ErrorNavigatorButton extends StatelessWidget {
   }
 }
 
+class InspectorDetailsOption extends StatelessWidget {
+  const InspectorDetailsOption({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return ValueListenableBuilder(
+      valueListenable: preferences.inspector.defaultInspectorDetailsView,
+      builder: (context, selection, _) {
+        final theme = Theme.of(context);
+        return Column(
+          children: [
+            Text(
+              'Select the tab which opens by default when opening the inspector.',
+              style: theme.subtleTextStyle,
+            ),
+            const SizedBox(
+              height: denseSpacing,
```

Comment:
style nit: no need for trailing comma on SizedBox widgets that only take one line.